reactioncommerce/reaction
# refactor: move system information plugin to npm

Impact: **minor**
Type: **refactor**

This PR removes the `system-info` plugin code from the API, and installs the `@reactioncommerce/plugin-system-information` to perform the same functions.

## Breaking changes
None

## Testing
The system information code is the same as it was before, it's just being installed from a different place, so you should see nothing different in the behavior of this PR and the `v3.1.0` release it was created from.
1. Start the API
1. Create a user
1. Create a shop
1. Use the `systemInformation` GraphQL query to query system information
```
{
systemInformation(shopId: "SHOP_ID") {
  plugins {
    version
    name
  }
  apiVersion
  mongoVersion {
    version
  }
}
}
```
1. See system information
